client/routes/search.js

```diff
@@ -1,13 +1,17 @@
 const log = require('../logger');
 const makeRequest = require('../makeRequest');
 const sendPacket = require('../sendPacket');
+const clickLogger = require('../clickLogger');
 
 const { User, Query } = require('../mongoConfig');
 
+var queryStartTime;
```

Comment:
define this variable inside the route. Doing this could cause issues with multiple users

Reply:
A better option if you need to access a variable in multiple routes, separated by user, is to store it in the request. Figure out how to set a web-token/cookie on the client